src/logging.cc

```diff
@@ -2566,3 +2559,20 @@ void DisableLogCleaner() {
 }
 
 _END_GOOGLE_NAMESPACE_
+
+void LogMessageTime::CalcGmtOffset() {
+  std::tm gmt_struct;
+  int isDst = 0;
+  if ( FLAGS_log_utc_time ) {
+    localtime_r(&timestamp_, &gmt_struct);
+    isDst = gmt_struct.tm_isdst;
+    gmt_struct = time_struct_;
+  } else {
+    isDst = time_struct_.tm_isdst;
+    gmtime_r(&timestamp_, &gmt_struct);
+  }
+
+  time_t gmt_sec = mktime(&gmt_struct);
+  gmtoffset_ = static_cast<long int>(timestamp_ - gmt_sec + (isDst ? 3600 : 0) ) ;
```

Comment:
Please add a comment what happens here: you're accounting for daylight saving time (+1 hour). Define the hour 3600 as a constant to avoid magic numbers.
```suggestion
  const long hour = 3600;
  gmtoffset_ = static_cast<long int>(timestamp_ - gmt_sec + (isDst ? hour : 0) ) ;
```

Reply:
Sure.